go/lib.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+ */
+
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+typedef enum RequestErrorType {
```

Comment:
Yes. Cbindgen can generate these bindings, but ORT rejects it since it is under the Mozilla Public License 2.0 (weak copyleft). uniffi-bindgen-go is under the same license and would also be rejected by ORT.

Reply:
I'll talk to the OSS org and see whether we can use CBindgen as a dev dependency or commandline tool.

Reply:
we've received permission to use MPL2 licensed code generating tools, as long as the tools themselves aren't a part of the package delivered to the user.